buildSrc/src/main/kotlin/util.kt

```diff
@@ -0,0 +1,153 @@
+import org.gradle.api.Action
+import org.gradle.api.artifacts.Dependency
+import org.gradle.api.artifacts.ExternalModuleDependency
+import org.gradle.api.artifacts.dsl.DependencyHandler
+import org.gradle.kotlin.dsl.DependencyHandlerScope
+import org.gradle.kotlin.dsl.accessors.runtime.addExternalModuleDependencyTo
+import org.gradle.nativeplatform.platform.internal.DefaultNativePlatform
+import java.net.URL
+import java.util.regex.Pattern
+
+// last commit of pom-scijava-29.2.1
+val pomSci = URL("https://raw.githubusercontent.com/scijava/pom-scijava/10b42e8e/pom.xml").readText()
```

Comment:
So this is the solution to using parent POMs in Gradle, nice! Can you make this more flexible so we can actually specify the POM version? Maybe you could get the XML actually from the Maven repo? E.g. from https://maven.scijava.org/content/groups/public/org/scijava/pom-scijava/29.2.1/pom-scijava-29.2.1.pom

Reply:
ah, that's much better actually, thanks

Reply:
Ok. I'll note that I sometimes use jitpack on pom-scijava versions so the commit hash fetching is kinda nice in that case. pom-scijava can have slow downs in the release process, which is why I started to do that. 

That said, it is also possible that maven.scijava.org's mirroring of jitpack would still work with the hash. Would need to check.

Reply:
Addressed